backend/src/core/database/utils/repository.py

```diff
@@ -0,0 +1,44 @@
+from abc import ABC, abstractmethod
+from uuid import UUID
+
+from sqlalchemy import select
+from sqlalchemy.dialects.postgresql import insert
+
+from backend.src.core.database.db import postgres_async_session
+
+
+class AbstractRepository(ABC):
+    @abstractmethod
+    async def get(self, *args, **kwargs):
+        raise NotImplementedError
+
+    @abstractmethod
+    async def all(self, *args, **kwargs):
+        raise NotImplementedError
+
+    @abstractmethod
+    async def create(self, *args, **kwargs):
+        raise NotImplementedError
+
+
+class SQLAlchemyRepository(AbstractRepository):
```

Comment:
Can you separate interfaces from implementation? It would be great to have separate folder for all interfaces